lib/common/components/buttons/custom_back_button.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_boilerplate/common/config/theme.dart';
+
+class CustomBackButton extends StatelessWidget {
+  final VoidCallback onPressed;
+  const CustomBackButton({Key? key, required this.onPressed}) : super(key: key);
+  @override
```

Comment:
there are two back button components (BackButton and CustomBackButton) are they both being used?